allennlp/training/callbacks/wandb.py

```diff
@@ -88,11 +88,7 @@ def __init__(
 
         self._watch_model = watch_model
         self._files_to_save = files_to_save
-
-        import wandb
-
-        self.wandb = wandb
-        self.wandb.init(
+        self._wandb_kwargs: Dict[str, Any] = dict(
```

Comment:
Nothing wrong with this, just `dict(...)` is kind of an unusual way of writing `{...}`.